components/BottomSheet/src/MDCBottomSheetController.h

```diff
@@ -55,6 +57,34 @@
  */
 @property(nonatomic, weak, nullable) id<MDCBottomSheetControllerDelegate> delegate;
 
+/**
+ The current state of the bottom sheet.
+ */
+@property(nonatomic, readonly) MDCSheetState state;
+
+/**
+ Sets the shape generator for state that is used to define the bottom sheet's shape for that state.
+
+ note: If a layer property is explicitly set after the shapeGenerator has been set,
+ it can lead to unexpected behavior.
+
+ When the shapeGenerator for a state is nil,
+ MDCBottomSheetController will use the default underlying layer with its default settings.
```

Comment:
nit: Kind of funky line breaking here.

Reply:
Could you expand a little with this documentation? Im not sure which layer properties I should not set explicitly after setting a shape generator, or which properties on a layer are the default settings relevant to the shape generator's behavior.